### PR TITLE
add planning application

### PR DESCRIPTION
**Pull request by ajwl**
Adds 2 new document types: Planning Applications and Council Petitions

### CODE REVIEW

File: common/src/main/scala/com/gu/localnews/common/services/index/CouncilPetitionsIndex.scala [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        "places" -> petition.entities.places,
+        "organisations" -> petition.entities.organisations,
+        "dates" -> petition.entities.dates, // TOOD make dates fields?
+        "keyPhrases" -> petition.entities.keyPhrases,
+        "sentiment" -> petition.entities.sentiment,
+      )
+    ) ++
+      petition.description.map("description" -> _) ++
+      petition.backgroundInfo.map("backgroundInfo" -> _) ++
+      petition.signatures.map("signatures" -> _)
```
Comment (ajwl) [inline common/src/main/scala/com/gu/localnews/common/services/index/CouncilPetitionsIndex.scala]: @itsibitzi this is coming out like:
```
"signatures": [
"PetitionSignature(Jordana  Yechiel)",
"PetitionSignature(Julia  Torr)",
"PetitionSignature(Pedro  Campos)",
"PetitionSignature(Andy  Comte-Lenestour)"
```
But it won't let me do `_.name` whyyyyyyy?

**Comment by ajwl** [comment]
Bug with signatures, otherwise i think this can go in...!

File: cli/src/main/scala/com/gu/localnews/cli/Main.scala [hunk 2]
```diff
@@ -14,15 +14,14 @@ object Main extends App {
...
-  // Setup first time
-  index.setupCouncilContracts()
 
   val options = new Args(args)
 
   options.`type`() match {
     case ImportType.CouncilContracts =>
+      // Setup first time IS THIS RIGHT?
+      val index = new Index(client)
+      index.setupCouncilContracts()
```
Comment (RichieAHB) [inline cli/src/main/scala/com/gu/localnews/cli/Main.scala:24]: We should only need to do this once at boot time (i.e. where it was) unless I'm misunderstanding!

**Review by RichieAHB — APPROVED** [review]
:+1 merge first - question later

File: cli/src/main/scala/com/gu/localnews/cli/Main.scala (hunk 2 above)
Comment (ajwl) [reply cli/src/main/scala/com/gu/localnews/cli/Main.scala:24]: @itsibitzi what's the correct way to do this?

**Pull request merged**